orchagent/orchdaemon.cpp

```diff
@@ -39,6 +39,10 @@ NatOrch *gNatOrch;
 
 bool gIsNatSupported = false;
 
+#define SWITCH_SHUTDOWN_LOG_INTERVAL_IN_SECS 60
+bool gIsSwitchShutdown = false;
```

Comment:
Suggest to place this new bool near line 40.

Reply:
done